src/EPR.PRN.Backend.Obligation.UnitTests/Strategies/MaterialCalculationStrategyResolverTests.cs

```diff
@@ -3,87 +3,88 @@
 using EPR.PRN.Backend.Obligation.Strategies;
 using Microsoft.Extensions.Logging;
 using Moq;
+using System.Diagnostics.CodeAnalysis;
 
-namespace EPR.PRN.Backend.Obligation.UnitTests.Strategies
+namespace EPR.PRN.Backend.Obligation.UnitTests.Strategies;
+
+[ExcludeFromCodeCoverage]
```

Comment:
remove

Reply:
We don't need to look for code coverage on unit tests